### PR TITLE
Improve position of verified icon in profile info

### LINKED ISSUE #3620: "green checkmark" looks detached in profiles
the "green checkmark" is usually shown directly beside the group or contact name, it is somehow regarded to be a part of it. 

however, this is not true for the group profiles - when the name is split to two rows, there is quite some random looking gap:

Screenshot 2024-01-06 at 13 27 47

(i also think, in this example, a wrap should not be needed at all)

if it is not possible for whatever reason to add the icon as a "inline", i would at least adapt the padding between text and icon to not more than 1em - and defer wrapping

### PR DESCRIPTION
**Pull request by adzialocha**
Inline verified icon with display name and center it with text when displaying profile info in groups and contacts.

Before:

2024-01-08-115901_608x312_scrot

After:

2024-01-08-115231_618x276_scrot

Bonus-Level:

* New `ProfileInfoHeader` component which is now reused across "view profile" and "view group" dialogs

Closes #3620

### CODE REVIEW

File: scss/dialogs/_group-styles.scss [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
-    overflow: hidden;
-    margin-bottom: 0;
-  }
-}
-
-input.group-name-input {
+  border: 0;
   border-bottom: solid;
   border-color: var(--loginInputFocusColor);
+  color: var(--bp4MenuText);
```
Comment (farooqkz) [inline scss/dialogs/_group-styles.scss:6]: Can we get rid of BP here? Our plan is to slowly get rid of BluePrint

File: src/renderer/components/Avatar.tsx [hunk 2]
```diff
@@ -94,10 +96,11 @@ export function AvatarFromContact(
...
 
-export function ClickForFullscreenAvatarWrapper(props: {
-  filename: string | null
-  children: React.ReactNode
-}) {
+export function ClickForFullscreenAvatarWrapper(
+  props: PropsWithChildren<{
+    filename?: string
+  }>
+) {
```
Comment (farooqkz) [inline src/renderer/components/Avatar.tsx:103]: Nice!

File: src/renderer/components/ProfileInfoHeader/index.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <div className={styles.profileInfoHeader}>
+      <ClickForFullscreenAvatarWrapper filename={avatarPath}>
+        <Avatar
+          displayName={displayName}
+          avatarPath={avatarPath}
+          color={color}
+          wasSeenRecently={wasSeenRecently}
+          large
+        />
+      </ClickForFullscreenAvatarWrapper>
```
Comment (farooqkz) [inline src/renderer/components/ProfileInfoHeader/index.tsx:35]: Nice pattern here! I continue to learn from you!

**Review by farooqkz — CHANGES_REQUESTED** [review]

File: scss/dialogs/_group-styles.scss (hunk 1 above)
Comment (Simon-Laux) [reply scss/dialogs/_group-styles.scss:6]: this is variable from us, many duplicate the var/color under a new name in themebase?

File: src/renderer/components/dialogs/ViewGroup.tsx [hunk 4]
```diff
@@ -247,22 +243,14 @@ function ViewGroupInner(
...
-                  {chat.isProtected && <InlineVerifiedIcon />}
-                </p>
-              </div>
+            <DialogContent paddingBottom>
+              <ProfileInfoHeader
+                avatarPath={groupImage ? groupImage : undefined}
+                color={chat.color}
+                displayName={groupName}
+                isVerified={chat.isProtected}
+                wasSeenRecently={chat.wasSeenRecently}
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/ViewGroup.tsx:252]: groups don't have wasSeenRecently (green dot)

**Review by Simon-Laux — APPROVED** [review]

File: src/renderer/components/dialogs/ViewGroup.tsx (hunk 4 above)
Comment (adzialocha) [reply src/renderer/components/dialogs/ViewGroup.tsx:252]: I'll make it optional to reflect that in the API

**Pull request merged**